workflows/offline_ml_diags/offline_ml_diags/_mapper.py

```diff
@@ -15,25 +15,37 @@
 
 
 class PredictionMapper(GeoMapper):
+    """A mapper of outputs from a predictor, with inputs coming from a base mapper."""
+
     def __init__(
         self,
         base_mapper: GeoMapper,
-        wrapped_model: fv3fit.Predictor,
+        predictor: fv3fit.Predictor,
         variables: Sequence[str],
         z_dim: str = "z",
-        rename_vars: Mapping[str, str] = None,
         grid: xr.Dataset = None,
     ):
+        """
+        Args:
+            base_mapper: mapper containing input data for prediction
+            predictor: model for prediction
+            variables: names of variables from base_mapper and grid to include in
+                input to predictive model, and in output datasets (alongside any
+                predicted outputs)
+            z_dim: name of the z-dimension used as a feature dimension
+            grid: constant dataset with prediction inputs
+        """
         self._base_mapper = base_mapper
-        self._model = wrapped_model
+        self._model = predictor
         self._z_dim = z_dim
+        # TODO: maybe split off responsibility of merging grid with each dataset
+        # in base_mapper into its own class, wrapping base_mapper
         self._grid = grid or xr.Dataset()
+        # TODO: can this arg be removed, instead using metadata on predictor?
```

Comment:
I just remembered the reason for this arg existing- the pressure thicknesses still need to be loaded so that 3D variables can be column integrated and have figures with pressure on one axis.

Reply:
If we decide to refactor, we could avoid this by merging the datasets outside of this object.